controller/api/models.py

```diff
@@ -299,6 +299,45 @@ def _start_containers(self, to_add):
         if set([c.state for c in to_add]) != set(['up']):
             err = 'warning, some containers failed to start'
             log_event(self, err, logging.WARNING)
+        # if the user specified a health check, try checking to see if it's running
+        try:
+            config = self.config_set.latest()
+            if 'HEALTHCHECK_URL' in config.values.keys():
+                # the app has 60 seconds to connect
+                num_retries = 60
+                for _ in xrange(num_retries):
```

Comment:
This could potentially loop 60 times with a 60s timeout. Should `num_retries` be used for both values here? An hour seems excessive, even if it's unlikely.